README.rst

```diff
@@ -6,71 +6,103 @@ Flake8 Spellcheck
 
 Flake8 Plugin that spellchecks variables, functions, classes and other bits of your python code.
 
-You can whitelist words that are specific to your project simply by adding them to ``whitelist.txt``
-in the root of your project directory. Each word you add  should be separated by a newline.
+You can whitelist words that are specific to your project simply by adding them to ``spellcheck-allowlist.txt``
```

Comment:
"You can whitelist" should be updated to "You can allowlist"

Reply:
"You can allow words" is probably more grammatically correct?

Reply:
Maybe. I think if whitelist was always considered grammatically correct, allowlist is equally as grammatically correct.

Reply:
True. But "you can whitelist..." should really have been "you can allow...." all along.

What if we re-wrote this as:
```
Words specific to your project are supported by adding them to ``spellcheck-allowlist.txt`` in the root of your project directory.
```

Reply:
looks good to me @shaleh 👍🏼